src/context/WalletProvider/WalletViewsSwitch.tsx

```diff
@@ -27,7 +27,7 @@ export const WalletViewsSwitch = () => {
   const translate = useTranslate()
   const match = useRouteMatch('/')
   const {
-    state: { wallet, modal, showBackButton, initialRoute, type },
+    state: { wallet, modal, showBackButton, initialRoute, onCloseRoute = '/', type },
```

Comment:
`onCloseRoute` is typed as a `string`, and so should never be `undefined`.

No need for the default value of the destructured property?